cinema/management/wait_for_db.py

```diff
@@ -0,0 +1,20 @@
+import time
+
+from django.core.management import BaseCommand
+from django.db import connections, OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database...")
+        connection = None
+        while not connection:
+            try:
+                connection = connections["default"]
+                connection.cursor().execute("SELECT 1")
```

Comment:
Executing a raw SQL query to check the database connection is not the best approach. Django provides a ready-to-use method `ensure_connection` that you can call on a database wrapper object. This method tries to establish a connection if it's not already established and returns `True` or `False` depending on whether it was successful.